genCode.py

```diff
@@ -175,9 +175,12 @@ def createStaticClient(name, api, genAsync=False):
 
 
     lines.append('    funcinfo = {')
-    items = functionInfo.iteritems() if six.PY2 else functionInfo.items()
-    for funcname, ref in sorted(items):
-        lines.append('        "%s": %s,' % (funcname, pprint.pformat(ref, indent=12)))
+
+    for funcname, ref in sorted(six.iteritems(functionInfo)):
```

Comment:
This looks sane, but would have been nice if auto-generated code changes were in a separate commit :)

Reply:
yah.... sorry about that.